config/settings.py

```
@@ -13,11 +13,11 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
 
-SECRET_KEY = os.environ.get("DJANGO_SECRET_KEY", None)
+SECRET_KEY = getenv("DJANGO_SECRET_KEY", None)
 if SECRET_KEY is None:
     raise ValueError("DJANGO_SECRET_KEY environment variable is not set")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)